docker/iqdb/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ruby:2.7.3-alpine
+
+ADD https://api.github.com/repos/zwagoth/iqdbs/git/refs/heads/master /tmp/iqdbs_version.json
+COPY iqdb.patch /tmp/iqdb.patch
+ENV IQDB_VERSION=20161008
+
+RUN apk --no-cache add build-base git libjpeg-turbo-dev gd-dev bash \
```

Comment:
Expected output?
```
g++ -c -o iqdb.o iqdb.cpp -O2  -DNDEBUG -Wall -DLinuxBuild -g  -DNO_SUPPORT_OLD_VER -DUSE_DELTA_QUEUE -DHAVE_UNORDERED_MAP -std=c++11 -fomit-frame-pointer -DFORCE_64BIT -DINTMATH -DLIB_GD -include stdint.h
make: gdlib-config: No such file or directory
make: gdlib-config: No such file or directory
g++ -c -o haar.o haar.cpp -O2  -DNDEBUG -Wall -DLinuxBuild -g  -DNO_SUPPORT_OLD_VER -DUSE_DELTA_QUEUE -DHAVE_UNORDERED_MAP -std=c++11 -fomit-frame-pointer -DFORCE_64BIT -DINTMATH -DLIB_GD -include stdint.h
make: gdlib-config: No such file or directory
g++ -c -o imgdb.o imgdb.cpp -O2  -DNDEBUG -Wall -DLinuxBuild -g  -DNO_SUPPORT_OLD_VER -DUSE_DELTA_QUEUE -DHAVE_UNORDERED_MAP -std=c++11 -fomit-frame-pointer -DFORCE_64BIT -DINTMATH -DLIB_GD -include stdint.h
make: gdlib-config: No such file or directory
g++ -c -o debug.o debug.cpp -O2  -DNDEBUG -Wall -DLinuxBuild -g  -DNO_SUPPORT_OLD_VER -DUSE_DELTA_QUEUE -DHAVE_UNORDERED_MAP -std=c++11 -fomit-frame-pointer -DFORCE_64BIT -DINTMATH -DLIB_GD -include stdint.h
make: gdlib-config: No such file or directory
g++ -c -o resizer.o resizer.cpp -O2  -DNDEBUG -Wall -DLinuxBuild -g  -DNO_SUPPORT_OLD_VER -DUSE_DELTA_QUEUE -DHAVE_UNORDERED_MAP -std=c++11 -fomit-frame-pointer -DFORCE_64BIT -DINTMATH -DLIB_GD -include stdint.h
/bin/sh: gdlib-config: not found
/bin/sh: gdlib-config: not found
g++ -o iqdb iqdb.o haar.o imgdb.o debug.o resizer.o   -lgd -ljpeg -lpng  -DNO_SUPPORT_OLD_VER -DUSE_DELTA_QUEUE -DHAVE_UNORDERED_MAP -std=c++11 -fomit-frame-pointer -DFORCE_64BIT -DINTMATH -DLIB_GD -include stdint.h
Cloning into '/iqdbs'...
```

Reply:
More or less yes. The config utility does not exist anymore in `libgd-dev` for years now I believe. Since it doesn't seem to have an effect on the output I just left it. Maybe there is a way to mute this message somehow.